exchange/adapter_builders.go

```diff
@@ -2,7 +2,7 @@ package exchange
 
 import (
 	"github.com/prebid/prebid-server/adapters"
-	"github.com/prebid/prebid-server/adapters/33across"
+	ttx "github.com/prebid/prebid-server/adapters/33across"
```

Comment:
Interested in what this ttx is supposed to represent, and why removing the SomoAudience adapter includes this update related to the 33across adapter?

Reply:
This is automatically inserted by Go's formatter since the package name doesn't match the directory name. Go has a convention of making the package name match the directory name, and when it doesn't this sort of stuff happens.